index.js

```diff
@@ -119,10 +122,12 @@ function fastifyWebsocket (fastify, opts, next) {
     // we always override the route handler so we can close websocket connections to routes to handlers that don't support websocket connections
     // This is not an arrow function to fetch the encapsulated this
     routeOptions.handler = function (request, reply) {
+      request.context[kWebSocketSchema] = request.context.schema ? fastify.validatorCompiler({ schema: request.context.schema.body }) : null
```

Comment:
Do not reuse body, but use a different property.

Reply:
If we're not using `body` property the issue is the validator will not be available because the fastify instance won't compute and expose it.

Reply:
Maybe we need to expose something more from Fastify to do it.

Reply:
Correct. I'll make a proposal in fastify